### PR TITLE
Deployment update

### PR DESCRIPTION
**Pull request by chrmod**
changes: 
* added SSM access
* terrafrom handles jenkins access

### CODE REVIEW

File: deploy/main.tf [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-  db_name = "balrog"
-  db_host = "balrog.cfheundkobhl.us-east-1.rds.amazonaws.com"
-  db_user = "balrogadmin"
-  db_pass = ""
+  key_name           = "sam"
+  vpc_id             = "vpc-2c11904a"
+  private_subnets    = ["subnet-2a058116", "subnet-9587f6ce", "subnet-6659712f", "subnet-bfafd492"]
+  public_subnets     = ["subnet-4004807c", "subnet-51587018", "subnet-8a84f5d1", "subnet-bda8d390"]
+  vpn_sec_group_id   = "sg-03d0a12d9dcc230eb"
+  ci_sec_group_id    = "sg-0998c98e562cd2f1c"
```
Comment (chrmod) [inline deploy/main.tf:10]: new security group for jenkins

File: deploy/main.tf [hunk 2]
```diff
@@ -71,4 +72,9 @@ EOF
 resource "aws_iam_role_policy_attachment" "telemetry-scheduler-ecr_access" {
   role       = aws_iam_role.balrog.name
   policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
-}
\ No newline at end of file
+}
+
+resource "aws_iam_role_policy_attachment" "balrog-web-ssm" {
```
Comment (chrmod) [inline deploy/main.tf:77]: ssm access

File: deploy/main.tf (hunk 1 above)
Comment (philipp-classen) [reply deploy/main.tf:10]: Maybe leave a comment what the configuration of this security group are expected to be. What permissions they grant.

BTW, security groups can be defined in Terraform as well. Perhaps that it is an option.

**Comment by chrmod** [comment]
covered by https://github.com/ghostery/balrog/pull/4

**Pull request closed without merging**